### ISSUE TITLE
HOT WHEELS UNLEASHED™ (1271700)

### ISSUE THREAD

**Issue by Teonyr** [ValveSoftware/Proton#5196]
# Compatibility Report
- Name of the game with compatibility issues: HOT WHEELS UNLEASHED™
- Steam AppID of the game: 1271700

## System Information
- GPU: NVIDIA GeForce RTX 3080
- Driver/LLVM version: Nvidia 470.74
- Kernel version: 5.14.9-300.fc35.x86_64
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Teonyr/e2f851d32da3d88c1da43a00bec2da77
- Proton version: Proton Experimental

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.

- [X] that I have checked whether there are updates for my system available.

## Symptoms:

Videos in the game don't seem to play, instead a tv test image is shown.

## Reproduction
1. Start the game.

[steam-1271700.log](https://github.com/ValveSoftware/Proton/files/7278638/steam-1271700.log)
![HotWheelsNoVideosPlaying](https://user-images.githubusercontent.com/75175693/135866757-e939e24d-a662-4165-a745-263bfcdd2478.png)

**Comment by kisak-valve** [comment]
Hot Wheels Unleashed - Out of video memory

Issue transferred from https://github.com/ValveSoftware/Proton/issues/5207.
@diegosandovalramirez posted on 2021-10-08T14:46:54:

# Compatibility Report
- Name of the game with compatibility issues: Hot Wheels Unleashed
- Steam AppID of the game:1271700

## System Information
- GPU:  GTX 1050 Ti
- Driver/LLVM version:  470.63.01
- Kernel version:  5.14.10-1
- Link to full system information report as [Gist](https://gist.github.com/diegosandovalramirez/caa60d617a024d3a231c3e84aee98a92):
- Proton version: Proton-6.18-GE-2

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.


[steam-1271700.log](https://github.com/ValveSoftware/Proton/files/7312038/steam-1271700.log)

## Symptoms 
Game exists with a wine message "Not enough memory for rendering resource" despite the game being fine in windows and the hardware being over the minnimum requirements.


## Reproduction
Runnning the game at any graphic setting will display this error. 
Depending on the graphic settings, the error will be displayed at different points of the game.
With my current setting it usaully happened at the first loop while at the top of it, now I got flipped so it crashed before. (Maybe a correlation with the camera being pointed up?)
![Screenshot_2021-10-08_11-41-48](https://user-images.githubusercontent.com/29217412/136577239-41f8970c-7007-4e04-8b9a-4fcd3015c494.png)

![Screenshot_2021-10-08_11-43-18](https://user-images.githubusercontent.com/29217412/136577249-2abc6e8f-0c40-426b-bc97-26a428066e57.png)

**Comment by kisak-valve** [comment]
Hello @diegosandovalramirez, the nVidia 470.63.01 driver release has a known regression that leads to high memory usage with vkd3d-proton and should be fixed with 470.74. Please retest with a newer video driver version once it becomes available via your package manager.

**Comment by diegosandovalramirez** [comment]
> Hello @diegosandovalramirez, the nVidia 470.63.01 driver release has a known regression that leads to high memory usage with vkd3d-proton and should be fixed with 470.74. Please retest with a newer video driver version once it becomes available via your package manager.

Thanks for the quick response. I will update the driver and see what happens. I will update here.

**Comment by Billli11** [comment]
add -dx11 to Launch option can run the game in dx11 mode.

**Comment by pierrep** [comment]
A recent game update has borked this game for me on Proton Experimental and latest Proton-GE. I can get it to run on 6.3.7 but obviously no videos work and it feels a bit glitchy at times. I haven't produced any logs yet, might look into it

**Comment by pierrep** [comment]
I just received another Proton experimental update, so I tried this again, and I have a log this time. Now after it fails I get a popup error that looks related to Unreal Engine.

[steam-1271700.log](https://github.com/ValveSoftware/Proton/files/7492138/steam-1271700.log)

**Comment by alvarlagerlof** [comment]
I can't get this to detect my Stadia controller, which does work in some other games like Dirt 3.

**Comment by pierrep** [comment]
@alvarlagerlof Proton experimental doesn't recognise any controllers in HWU at this point. It now runs but is keyboard only. Proton GE doesn't run, it's just a black screen. You can play the game with latest stable 6.3.7 or 6.3.8, but sometimes it doesn't recognise controllers and you need to restart. If you want to play coop, sometimes it will freeze when you hit a controller button, and you'll need to restart. It eventually works with a controller after 1-3 restarts. This game worked flawlessly previously with experimental, and the videos worked too. It's not Valve's fault, the game's Devs updated the game, with no perceivable improvement, and now it's flaky. Once you get in game though and controllers work, it's solid, for the most part.

**Comment by alvarlagerlof** [comment]
**@pierrep ** My I'll try with non-expert mental. Currently I'm forced to use proton starting on 5.x.x due to the latency bug so I can't haven anything recently anyways.

**Comment by ruineka** [comment]
I wanted to chime in here and say that I've been playing Hot wheels Unleashed for a few hours now using Proton experimental with no issues at all, using controllers and keyboard/mouse, even been gamestreaming it. Rock solid experience.

AMD 6700 XT
i7 7700k
Pop OS 21.04

using -dx11 as a launch option

**Comment by pierrep** [comment]
@ruineka yes it improved dramatically again recently, not sure if it was a Proton or a game update. I still rarely get a black screen on start up but generally it's stable now. Why the -dx11 start up flag? Isn't it dx11 only?

**Comment by ruineka** [comment]
> @ruineka yes it improved dramatically again recently, not sure if it was a Proton or a game update. I still rarely get a black screen on start up but generally it's stable now. Why the -dx11 start up flag? Isn't it dx11 only?

I'm assuming its both dx12 and 11. Without the dx11 launch option the game crashes shortly after the start screen, if its a fresh gamesave it crashes right when you attempt to open a loot box thing on Intel Iris Xe.

On a GTX 1080/AMD 6700 XT the launch option is not required it seems.

**Comment by ruineka** [comment]
Using `DX12` the game loads into the initial menu, but once you try to load into a track the game locks up on a `Intel A750 Arc GPU`. 

Running with `Mesa 22.3.1 - kisak-mesa` and kernel `6.0.6-76060006-generic` on Pop OS.
[steam-1271700.log](https://github.com/ValveSoftware/Proton/files/10282944/steam-1271700.log)

**Comment by pierre-aimi** [comment]
Recently I discovered that the Split-screen mode is broken. When you connect your two controllers, the game freezes. This game is one of the most unstable games I have in my collection. When it runs, it's fine. If it runs.

**Comment by PrinceMerluza** [comment]
Just got a Steam Deck and bought and installed this. Latest proton. Can't run the game because it says it needs Microsoft C++ redist. Is Proton supposed to handle these dependencies

**Comment by kisak-valve** [comment]
Hot Wheels Unleased (1271700)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6666.
@Lord-Krex posted on 2023-04-07T12:30:20:

# Compatibility Report
- Name of the game with compatibility issues: Hot Wheels Unleased
- Steam AppID of the game: 1271700

## System Information
- Steam Deck
- Proton version: 7.0-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 

This is a verified title. Menu screen and boot screen are messed up / rainbow colors and split screen doesn't seem to work. Seems to be an issue with it loading certain assists.

![hotwheels](https://user-images.githubusercontent.com/130154278/230609030-1c03c63b-9a67-4a66-ba44-e5981edf1374.jpg)

## Reproduction

1. Download game to Steam Deck.
2. Play game.

**Comment by alasky17** [comment]
@Lord-Krex Thank you for the feedback!  It looks like this one missed getting tagged as requiring h264.  We have improved the system to avoid missing such things, but this title was verified before the improvements.  Until this gets fixed globally, you can enable h264 manually by switching to desktop mode and running steam steam://unlockh264/ in a terminal and then rebooting your deck.
